### PR TITLE
ENH: Add physical dimensions to forbild phantom, closes #805

### LINKED ISSUE #805: Allow users to specify forbild phantom in physical dimensions
The forbild phantom is currently rescaled so that the domain is assumed to be [-12.8, 12.8] x [-12.8, 12.8]. In some cases users could however want to use the actual dimensions of their space (i.e. if their space is [-15, 15] x [-15, 15], they may want to have a bit more "air" around it).

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: odl/phantom/transmission.py [hunk 1]
```diff
@@ -287,6 +288,16 @@ def forbild(space, resolution=False, ear=True, value_type='density'):
...
+    scale : {'auto', 'cm', 'meters', 'mm'}
+        Controls how ``space`` should be rescaled to fit the definition of
+        the forbild phantom, which is defined on the square
+        [-12.8, 12.8] x [-12.8, 12.8] cm.
+
+        * 'auto' means that space is rescaled to fit exactly. The space is also
+          centered at [0, 0].
+        * 'cm' means the dimensions of the space should be used as is.
+        * 'meters' means all dimensions of the space are multiplied by 100.
+        * 'mm' means all dimensions of the space are divided by 10.
```
Comment (kohr-h) [inline odl/phantom/transmission.py]: Can you put the strings between double backticks?

File: odl/phantom/transmission.py [hunk 2]
```diff
@@ -287,6 +288,16 @@ def forbild(space, resolution=False, ear=True, value_type='density'):
...
         'materials' returns indices in the range [0, 7].
+    scale : {'auto', 'cm', 'meters', 'mm'}
+        Controls how ``space`` should be rescaled to fit the definition of
+        the forbild phantom, which is defined on the square
+        [-12.8, 12.8] x [-12.8, 12.8] cm.
+
+        * 'auto' means that space is rescaled to fit exactly. The space is also
+          centered at [0, 0].
+        * 'cm' means the dimensions of the space should be used as is.
+        * 'meters' means all dimensions of the space are multiplied by 100.
```
Comment (kohr-h) [inline odl/phantom/transmission.py]: For consistency, `'m'`

File: odl/phantom/transmission.py [hunk 3]
```diff
@@ -317,10 +328,23 @@ def transposeravel(arr):
...
+    if scale == 'auto':
+        xcoord = ((xcoord - space.min_pt[0]) /
+                  (space.max_pt[0] - space.min_pt[0]))
+        xcoord = 25.8 * xcoord - 12.8
+        ycoord = ((ycoord - space.min_pt[1]) /
+                  (space.max_pt[1] - space.min_pt[1]))
+        ycoord = 25.8 * ycoord - 12.8
+    elif scale == 'cm':
+        pass  # dimensions already correct.
+    elif scale == 'meters':
```
Comment (kohr-h) [inline odl/phantom/transmission.py]: comment above applies here, too

**Comment by adler-j** [comment]
Updated by comments, merge?

**Comment by kohr-h** [comment]
Do it!

**Pull request merged**